.github/workflows/wheels.yml

```diff
@@ -43,18 +47,52 @@ jobs:
         fetch-tags: true
         fetch-depth: 0
 
-    - uses: pypa/cibuildwheel@v2.19.2
+    - uses: pypa/cibuildwheel@v2.22
       env:
+        CIBW_ARCHS: ${{ matrix.arch }}
         MACOSX_DEPLOYMENT_TARGET: 10.14
 
     - name: Upload wheels
       uses: actions/upload-artifact@v4
       with:
-        name: artifact-wheel-${{ matrix.os }}
+        name: artifact-wheel-${{ matrix.os }}-${{ matrix.arch }}
         path: wheelhouse/*.whl
 
+  build_alt_wheels:
+    name: Wheel on ${{ matrix.os }} (cp${{ matrix.python }}/${{ matrix.libc }}/${{ matrix.arch }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        libc: [manylinux, musllinux]
+        arch: [aarch64]
+        python: [37, 38, 39, 310, 311, 312, 313]
```

Comment:
Since we aren't testing py37 I'm inclined to not build wheels for it either

Reply:
the x86 wheels for other OSs were building 3.7 based on `python_requires >= 3.7` so I matched that here.
I've bumped that up to 3.8.